content/javascript/concepts/loops/loops.md

```diff
@@ -159,6 +159,24 @@ for (let i = 0; i < array.length; i++) {
 // Output: Every item in the array
 ```
 
+## Looping Through Objects
+
+To loop through objects use `Object.entries` which will first convert the objects into an array and then loop through that array. 
+
+Using `Object.entries` an array of arrays is created which will contain both the property and the value respectively.
+
+```js
+const objectK = {
+  name: 'Codecademy',
+  age: 10
+};
+```
+The output would look like:
+``` 
```

Comment:
To keep consistent with the rest of the output snippets in this entry, let's change this to ```shell

Reply:
Alright!